src/main/java/seedu/address/logic/commands/ListTagsCommand.java

```diff
@@ -34,13 +38,13 @@ public CommandResult execute(Model model, CommandHistory history) {
 
         StringBuilder stringBuilder = new StringBuilder();
 
-        Set<Tag> tagSet = model.getHumanPlayer().getFleet().getAllTags();
+        tagSet = model.getHumanPlayer().getFleet().getAllTags();
         if (tagSet.isEmpty()) {
-            stringBuilder.append("no tags found");
+            stringBuilder.append(TAGS_NOT_FOUND);
```

Comment:
Using a static final string here is better than inputting the string contents manually as it can be managed easily at the start of the file. Nice!

Reply:
Yes, I am trying to avoid the use of magic values.